docs/develop/build-a-dApp/build-smart-contracts/evm_extensions.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 3
+---
+
+# EVM Extensions
+
+## Overview
+
+EVM Extensions are precompiled contracts that are built into the Ethereum
+Virtual Machine (EVM).
+Each offers specific functionality, that can be used by other smart contracts.
+Generally, they are used to perform operations that are either not possible
+or would be too expensive to perform with a regular smart contract
+implementation, such as hashing, elliptic curve cryptography, and modular exponentiation.
+
+By adding custom EVM extensions to Ethereum's basic feature set,
+Evmos allows developers to use previously unavailable functionality in smart contracts, like staking and governance operations.
+This will allow more complex smart contracts to be built on Evmos and further improves the interoperability between Cosmos and Ethereum.
+It also is a key feature to achieve Evmos' vision of being the definitive dApp
+chain, where any dApp can be deployed once and users are able to interact with
+a wide range of different blockchains natively.
+
+## Available Extensions
+
+The following extensions are available in Evmos' implementation of the EVM:
+
+| Address                                      | Name            | Stateful | EIP                                               |
```

Comment:
Would be important to explain what stateful vs nonstateful means. Its also nice to understand how its used and why it is important here.

Reply:
stateful = performs a state transition (transaction)